cmd/skaffold/app/cmd/flags.go

```diff
@@ -227,6 +227,15 @@ var FlagRegistry = []Flag{
 		FlagAddMethod: "BoolVar",
 		DefinedOn:     []string{"dev", "run"},
 	},
+	{
+		Name:          "output",
+		Shorthand:     "o",
+		Usage:         "Writes '--render-only' output to the specified file",
+		Value:         &opts.Output,
+		DefValue:      "",
+		FlagAddMethod: "StringVar",
+		DefinedOn:     []string{"dev", "run"},
```

Comment:
does this flag really need to be defined on `dev` as well?

Reply:
fixed.